Assets/Scripts/Door.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Door : Entity
+{
+    [Header("Destructible From: ")]
+    [SerializeField] bool TheLeft = false;
+    [SerializeField] bool TheRight = false;
+    [SerializeField] bool Up = false;
+    [SerializeField] bool Down = false;
+
+
+    [EasyButtons.Button]
+    public override void TakeDamage(float baseDamage, Vector2 direction)
+    {
+
+        if ((TheLeft && direction.x < 0)|| (TheRight && direction.x > 0))
```

Comment:
I'm not sure why there are two if statements. correct me if I'm wrong but I think it would work in one if statement.